tests/test_raster.py

```diff
@@ -65,7 +66,7 @@ def test_dimorder_error(self):
         with self.assertRaises(TypeError):
             Image(TEST_FILE, dimorder="middle")
 
-    def test_arr(self):
+    def test_arr_dimorder_first(self):

```

Comment:
I would say it is the other way round, the first two are initialized with a file, the last two with an array.